erdjs-snippets/lending-pool/priceAggregatorPoolInteractor.ts

```diff
@@ -0,0 +1,134 @@
+
+        console.log(`PriceAggregatorInteractor.deployAggregator(): contract = ${address}`);
+        return { address, returnCode };
+    }
+
+    async unpausePoolAggregator(user: ITestUser) {
+        // console.log(`PriceAggregatorInteractor.unpausePoolAggregator(): address = ${user.address}`);
+
+        // Prepare the interaction
+        let interaction = <Interaction>this.contract.methods
+            .unpause([])
+            .withGasLimit(10000000)
+            .withNonce(user.account.getNonceThenIncrement())
+            .withChainID(this.networkConfig.ChainID);
+
+        // Let's check the interaction, then build the transaction object.
+        let transaction = interaction.check().buildTransaction();
+
+        // Let's sign the transaction. For dApps, use a wallet provider instead.
+        await user.signer.sign(transaction);
+
+        // Let's broadcast the transaction and await its completion:
+        await this.networkProvider.sendTransaction(transaction);
+        let transactionOnNetwork = await this.transactionWatcher.awaitCompleted(transaction);
+
+        // In the end, parse the results:
+        let { returnCode } = this.resultsParser.parseOutcome(transactionOnNetwork, interaction.getEndpoint());
+        return returnCode;
+    }
+
+    async submitPriceAggregator(user: ITestUser, from: string, to: string, price: number) {
+        console.log(`PriceAggregatorInteractor.submitPriceAggregator(): address = ${user.address}, from = ${from}, to = ${to}, price = ${price}`);
+
+        // Prepare the interaction
+        let interaction = <Interaction>this.contract.methods
+            .submit([from, to, price])
+            .withGasLimit(10000000)
+            .withNonce(user.account.getNonceThenIncrement())
+            .withChainID(this.networkConfig.ChainID);
+
+        // Let's check the interaction, then build the transaction object.
+        let transaction = interaction.check().buildTransaction();
+
+        // Let's sign the transaction. For dApps, use a wallet provider instead.
+        await user.signer.sign(transaction);
+
+        // Let's broadcast the transaction and await its completion:
+        await this.networkProvider.sendTransaction(transaction);
+        let transactionOnNetwork = await this.transactionWatcher.awaitCompleted(transaction);
+    }
+
+    async latestPriceFeed(from: string, to: string) {
+        // Prepare the interaction, check it, then build the query:
+        let interaction = <Interaction>this.contract.methods.latestPriceFeed([from, to]);
+        let query = interaction.check().buildQuery();
+
+        // Let's run the query and parse the results:
+        await this.networkProvider.queryContract(query);
+        let queryResponse = await this.networkProvider.queryContract(query);
+        let { values } = this.resultsParser.parseQueryResponse(queryResponse, interaction.getEndpoint());
+
+        // Now let's interpret the results.
```

Comment:
The comment says something about interpreting the results. Perhaps remove comment?